doc/walkthrough.md

```diff
@@ -25,23 +25,23 @@ If a SQL statement is of the standard syntax, SQLFlow throws it to the SQL engin
 
 ### SQLFlow as a gRPC Server
 
-SQLFlow is a gRPC server, which can connect with multiple clients.  A typical client is [pysqlflow](https://github.com/sql-machine-learning/pysqlflow), the SQLFlow plugin for Jupyter Notebook server.  Another once is a text-based client [/cmd/sqlflowserver/main.go](/cmd/sqlflowserver/main.go).
+SQLFlow is a gRPC server, which can connect with multiple clients.  A typical client is [pysqlflow](https://github.com/sql-machine-learning/pysqlflow), the SQLFlow plugin for Jupyter Notebook server.  Another once is a text-based client [/cmd/sqlflowserver/main.go](https://github.com/sql-machine-learning/sqlflow/tree/develop/cmd/sqlflowserver/main.go).
```

Comment:
Better to use relative paths? Former code redirect from `/` so that it was not working I guess.

Reply:
Relative paths work on the markdown file but won’t work on the actual website since the source code is not available under sqlflow website. It depends on how the website is set up. As I mentioned in the description, this should be a temporary solution so at least no 404 is returned.